mava/networks.py

```diff
@@ -151,6 +155,42 @@ def __call__(self, obs_embedding: chex.Array, observation: Observation) -> tfd.I
         )
 
 
+class DiscreteActionEpsGreedyMaskedHead(nn.Module):
```

Comment:
Discrete action is implied in my opinion

```suggestion
class MaskedEpsilonGreedyHead(nn.Module):
```